### PR TITLE
Extend API Config

### PR DESCRIPTION
**Pull request by jumaallan**
Story: https://app.shortcut.com/smileid/story/13339/extend-api-config-initialise-function-on-cross-platform-wrappers

## Summary

Merge after #96 

## Known Issues

Any shortcomings in your work. This may include corner cases not correctly handled or issues related
to but not within the scope of your PR. Design compromises should be discussed here if they were not
already discussed above.

## Test Instructions

Concise test instructions on how to verify that your feature works as intended. This should include
changes to the development environment (if applicable) and all commands needed to run your work.

## Screenshot

If applicable (e.g. UI changes), add screenshots to help explain your work.

### CODE REVIEW

File: android/src/main/kotlin/com/smileidentity/flutter/SmileIDPlugin.kt [hunk 1]
```diff
@@ -83,12 +84,32 @@ class SmileIDPlugin : FlutterPlugin, SmileIDApi, ActivityAware {
...
-        SmileID.initialize(context = appContext, enableCrashReporting = false)
+    override fun initializeWithApiKey(
+        apiKey: String,
+        config: FlutterConfig,
+        useSandbox: Boolean,
+        enableCrashReporting: Boolean,
+    ) {
+        SmileID.initialize(
+            context = appContext,
+            apiKey = apiKey,
```
[the remark this answers is not in the document]
Comment (jumaallan) [reply android/src/main/kotlin/com/smileidentity/flutter/SmileIDPlugin.kt:95]: sadly yes, we need the api url from config still
